src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```
@@ -783,26 +624,20 @@ public function theLastOrderShouldHavePaymentState(string $orderPaymentState): v
         Assert::true($this->indexPage->isSingleResourceOnPage(['paymentState' => $orderPaymentState]));
     }
 
-    /**
-     * @Then the order :order should have order shipping state :orderShippingState
-     * @Then /^(this order) should have order shipping state "([^"]+)"$/
-     */
+    #[Then('the order :order should have order shipping state :orderShippingState')]
+    #[Then('/^(this order) should have order shipping state "([^"]+)"$/')]
     public function theOrderShouldHaveShippingState(OrderInterface $order, string $orderShippingState): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$order'. (undefined)

(UnusedFormalParameter)